hack/utils/dupissue/dupissue.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+const (
+	// TemplateFullPathFilename filename
```

Comment:
Is this commented out code needed?

Reply:
it's still being used. you can provide your own custom template and the search order is:
- next to he binary
- homedir
- or just use the default (contained in the source code)

Reply:
👍🏼